wire/core/WireMail.php

```diff
@@ -328,11 +328,12 @@ public function param($value) {
 	 * @return this 
 	 *
 	 */
-	public function attachment($value) {
+	public function attachment($value, $filename = '') {
 		if(is_null($value)) {
 			$this->mail['attachments'] = array();
 		} else if(is_file($value)) { 
-			$this->mail['attachments'][] = $value; 
+			$filename = $filename ?: basename($value);
+			$this->mail['attachments'][$filename] = $value; 
```

Comment:
True. I've no hard feelings about that, so if @ryancramerdesign thinks switching would be the better call I'm fine with it.